src/Carousel.js

```diff
@@ -269,6 +288,7 @@ class Carousel extends React.Component {
           onClick={this.handlePrev}
         >
           {prevIcon}
+          {prevLabelNode}
```

Comment:
should this be an explicit `null` check? what behavior do we want if the user sets `prevLabel=""`?